bugsnag-android-core/src/main/java/com/bugsnag/android/Delivery.kt

```diff
@@ -57,9 +57,9 @@ interface Delivery {
      * See [https://docs.bugsnag.com/api/error-reporting/]
      * (https://docs.bugsnag.com/api/error-reporting/)
      *
-     * @param report         The error report
+     * @param eventPayload         The error eventPayload
      * @param deliveryParams The delivery parameters to be used for this request
      * @return the end-result of your delivery attempt
      */
-    fun deliver(report: Report, deliveryParams: DeliveryParams): DeliveryStatus
+    fun deliver(eventPayload: EventPayload, deliveryParams: DeliveryParams): DeliveryStatus
```

Comment:
Perhaps a simpler parameter name? (Also other functions with same parameter..)

```suggestion
    fun deliver(payload: EventPayload, deliveryParams: DeliveryParams): DeliveryStatus
```